easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.0-intel-2017b-Python-3.6.3.eb

```diff
@@ -16,8 +16,9 @@ exts_defaultclass = 'PythonPackage'
 
 dependencies = [
     ('Python', '3.6.3'),
+    ('Tkinter', '%(pyver)s', versionsuffix),
     ('libpng', '1.6.32'),
-    ('freetype', '2.8.1'),
+    ('freetype', '2.8'),
```

Comment:
@schiotz Please stick to `('freetype', '2.8.1')` here, to avoid introducing conflicts.

Reply:
As above :-)